articles/flows/guides/implicit/add-login-implicit.md

```diff
@@ -16,15 +16,17 @@ useCase:
 # Add Login Using the Implicit Flow
 
 ::: note
-This tutorial will help you add login to your single-page application (SPA) using the Implicit Flow. If you want to learn how the flow works and why you should use it, see [Implicit Flow](/flows/concepts/implicit). If you want to learn to call your API from a SPA, see [Call Your API Using the Implicit Flow](/flows/guides/implicit/call-api-implicit).
```

Comment:
Considering their isn't a good use case for a SPA to only Authenticate a user without requesting an AT as well, I think the best option for this doc is to kill it and instead have the link for Login -> Single-Page App link to the same document that Login -> Native App

As a follow up, we should create a ticket to create a page that helps a user decide if their SPA is a Public or Private Client and then point them to either the "Add Login with Code Flow" or the "Add login with Code Flow + PKCE"

Reply:
This is supposed to be the guide for how to implement getting an ID Token to authenticate using Implicit Flow. So if we're presenting Implicit Flow w/ Form Post as an option, shouldn't this be rewritten to match that implementation rather than killed?

Reply:
Going to open a different PR with edits to our "Choose your Flow" page and ask you for a separate review on that.